celest/coordinates/ground_location.py

```diff
@@ -0,0 +1,171 @@
+
+
+from celest.angle_strings import _ISO6709_representation
+from celest.constants import WGS84_MINOR_AXIS_KM, WGS84_MAJOR_AXIS_KM
+from celest.units.quantity import Quantity
+from celest.units.core import Unit
+from celest import units as u
+from math import cos, sin, sqrt
+
+
+class GroundLocation:
+    """GroundLocation(latitude, longitude, height, angular_unit, length_unit)
+
+    Specify Earth bound location.
+
+    The `GroundLocation` class models the Earth with the WGS84 reference
+    ellipsoid to provide accurate representations of ground locations.
+
+    Parameters
```

Comment:
The latitude and longitude parameters do not have to be degrees like the documentation states. Update the documentation.

Reply:
They do; the inputs are floats.